docs/root/configuration/overview/v2_overview.rst

```diff
@@ -0,0 +1,330 @@
+.. _config_overview_v2:
+
+Overview (v2 API)
+=================
+
+The Envoy v2 APIs are defined as `proto3
+<https://developers.google.com/protocol-buffers/docs/proto3>`_ `Protocol Buffers
+<https://developers.google.com/protocol-buffers/>`_ in the `data plane API
+repository <https://github.com/envoyproxy/data-plane-api/tree/master/api>`_. They evolve the
+existing :ref:`v1 xDS APIs and concepts <config_overview_v1>` to support:
+
+* Streaming delivery of xDS API updates via gRPC. This reduces resource requirements and can
+  lower the update latency.
+* A new REST-JSON API in which the JSON/YAML formats are derived mechanically via the `proto3
+  canonical JSON mapping
+  <https://developers.google.com/protocol-buffers/docs/proto3#json>`_.
+* Delivery of updates via the filesystem, REST-JSON or gRPC endpoints.
+* Advanced load balancing through an extended endpoint assignment API and load
+  and resource utilization reporting to management servers.
+* `Stronger consistency and ordering properties
+  <https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md#eventual-consistency-considerations>`_
+  when needed. The v2 APIs still maintain a baseline eventual consistency model.
+
+See the `xDS protocol description <https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md>`_ for 
+further details on aspects of v2 message exchange between Envoy and the management server.
+
+Bootstrap configuration
+-----------------------
+
+To use the v2 API, it's necessary to supply a bootstrap configuration file. This
+provides static server configuration and configures Envoy to access :ref:`dynamic
+configuration if needed <arch_overview_dynamic_config>`. As with the v1
+JSON/YAML configuration, this is supplied on the command-line via the ``-c``
+flag, i.e.:
+
+.. code-block:: console
+
+  ./envoy -c <path to config>.{json,yaml,pb,pb_text}
+
+where the extension reflects the underlying v2 config representation.
+
+The :ref:`bootstrap configuration proto <envoy_api_file_api/bootstrap.proto>` describes
+the static config file format.
+
+The :ref:`Bootstrap <envoy_api_msg_Bootstrap>` message is the root of the
+configuration. A key concept in the :ref:`Bootstrap <envoy_api_msg_Bootstrap>`
+message is the distinction between static and dynamic resouces.  Resources such
+as a :ref:`Listener <config_listeners>` or :ref:`Cluster
+<config_cluster_manager_cluster>` may be supplied either statically in
+``static_resources`` or have an xDS service such as :ref:`LDS
```

Comment:
nit: per the new contribution guide I would make these italics (assuming we agree on that) or better direct links to the fields. Same below.

Reply:
I probably have a slight preference for inline quotes, but it's arbitrary and consistency is more important, so will go with italics or links.